packages/engine/vscode/src/lib/compile_worker.ts

```
@@ -0,0 +1,118 @@
+import * as solc from "solc";
+import * as path from "path";
+import * as fs from "fs";
+import axios from "axios";
+import { RemixURLResolver } from "remix-url-resolver";
+
+
+function handleLocal(pathString: string, filePath: any) {
+	// if no relative/absolute path given then search in node_modules folder
+	if (pathString && pathString.indexOf(".") !== 0 && pathString.indexOf("/") !== 0) {
+		console.error("Error: Node Modules Import is not implemented yet!");
```

Maybe throw an error instead ?